server/handler.py

```diff
@@ -123,9 +123,13 @@ def send_email(submission_record, submission_id):
     )
     email_body = (
         f"<body>"
+        f"<p>Thank you for providing your student debt story to the Two Cent Project! For security purposes, we must verify your email before publishing your story on our website.</p>"
```

Comment:
`the Two Cent Project` -> `Two Cent Stories`

sounds more friendly

Reply:
`For security purposes, we must verify your email before publishing your story on our website.` -> `For security purposes, we'd like to verify your email before publishing your story on our website.`